logIn/templates/logIn/index.html

```diff
@@ -38,6 +38,7 @@ <h4 class="h5 mb-3 font-weight-light">to continue, login with your</h4>
                     <p class="errorlist" style="color: red">Invalid credentials</p>
                 {% endif %}
                 <input class="btn btn-lg btn-primary btn-block" type="submit" value="Login"/>
+                <a style="margin:auto;" href="{% url 'changepass:reset_password' user_type %}">Forgot Password?</a>

```

Comment:
Having "class=mt-3" would give this link a bit of top margin and separate it from the login button, "margin: auto" seems to not put any margin to this link rn,

Reply:
changing it to have class does not seem to add any extra margin